proposals/stf-general-investment.md

```diff
@@ -0,0 +1,263 @@
+# The great fix forward – Sovereign Tech Fund general investment
+
+This proposal was collaboratively developed by
+
+- @fricklerhandwerk
+- @proofconstruction
+- @RaitoBezarius
+- @wamirez
+
+with contributions by (in ASCIIbetical order):
+
+- @Ericson2314
+- @Janik
+- @edef
+- @mweinelt
+- @refroni
+- @roberth
+- @tfc
+
+## Project title
+
+*May be identical with application name*
+
+Transforming global software distribution with Nix
+
+## Describe your project in a sentence. 100 words
+
+Nix makes computation with file systems highly repeatable; Nixpkgs is the largest, most up-to-date free software collection in the world; NixOS is a Linux distribution with a uniform declarative configuration interface.
+
+## Describe your project more in-depth. Why is it critical? 300 words
+
+[Nix](https://nix.dev) reduces the global cost of software production and long-term use by a tangible amount.
+It is the basis of an ecosystem of exceptionally powerful tools – including Nixpkgs, [the largest, most up-to-date software repository in the world](https://repology.org/repositories/graphs), and NixOS, a Linux distribution that can be configured fully declaratively, with unmatched flexibility.
+
+The Nix ecosystem is already on the way to become a default in the software development tool stack, by drastically shortening the time required to start working on software projects – written in any language – and lowering the cost of maintaining and sharing build setups and development environments.
```

Comment:
A few testimonials as "sources" would be great here, i.e. "big" companies such as Target or Shopify using Nix as the default tool for dev envs.

Reply:
We have a few exemplary links in the "projects depending on Nix" section. Also AFAIK Shopify stopped using Nix, and we seem to have no one contact at Target.

Reply:
More significantly, it's an important or fundamental compoennt of PaaS like Google IDX and Replit.

I'd add Hercules CI if it wasn't still in the early adopter stage.